st2api/tests/unit/controllers/v1/test_executions_simple.py

```diff
@@ -385,27 +385,28 @@ def test_post_parameter_validation_explicit_none(self):
     def test_post_with_st2_context_in_headers(self):
         resp = self._do_post(copy.deepcopy(LIVE_ACTION_1))
         self.assertEqual(resp.status_int, 201)
-        parent_user = resp.json['context']['user']
-        parent_exec_id = str(resp.json['id'])
+        parent_user = unicode(resp.json['context']['user'])
```

Comment:
Would need to research but perhaps safer and more cross-compatible to use `st2common.util.compat.to_unicode` (just using `unicode` won't work with all Python 2 and 3 versions.